### DISCUSSION TITLE
How to change the e.g. borderradius on the fly based on a database entry

### DISCUSSION THREAD

**Discussion by sven09** [tailwindlabs/tailwindcss#3048] [Help]
Hi, 

Scenario: We would like to change/disable the borderradius for elements like containers and buttons (etc.) based on a flag in the backend.

What is the best way to change borderradius on the fly for all items that have a certain class?
I think it can work similar to the dark mode feature:
(https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually)

```

<!-- NotRoundedMode not enabled -->
<html>
<body>
  <!-- Will be rounded -->
  <div class="rounded NotRoundedMode:rounded-none">
    <!-- ... -->
  </div>
</body>
</html>

<!--  NotRoundedMode enabled -->
<html class="NotRoundedMode">
<body>
  <!-- Will be not rounded -->
    <div class="rounded NotRoundedMode:rounded-none">
    <!-- ... -->
  </div>
</body>
</html>
```

Is this the best approach?

Any hint would be very helpful.

Many thanks.

**Comment by simonswiss** [comment]
Hey!

That would work with a custom `variant` if you want it to work without needing any JavaScript.

Another solution would be to use a little JS to dynamically toggle the `rounded` class on various elements.
